Quadriga/src/main/java/edu/asu/spring/quadriga/aspects/DictionaryAuthorization.java

```diff
@@ -53,6 +49,9 @@ public boolean chkAuthorization(String userName, String accessObjectId,
         IDictionary dictionary = dictonaryManager
                 .getDictionaryDetails(accessObjectId);
 
+        if(dictionary == null){
+        	return false;

```

Comment:
I think you need to throw an exception here like for workspaces. Or am I misreading this? If someone tries to access a dictionary that doesn't exist than that is an error not authorization denied.